src/sentry/roles/manager.py

```diff
@@ -200,3 +200,10 @@ def with_any_scope(self, scopes: Iterable[str]) -> Iterable[OrganizationRole]:
 
     def get_minimum_team_role(self, org_role: str) -> TeamRole:
         return self.team_roles.get(self._minimum_team_role_map[org_role])
+
+    def get_sorted_organization_roles(self, org_roles: Iterable[str]) -> Iterable[OrganizationRole]:
```

Comment:
The methods on `RoleManager` that deal only with one level of role are a legacy thing. I would rather put this method on `RoleLevel` (and change the names to not refer to org roles, of course).

Then the calls would be `organization_roles.get_sorted_roles` rather than `roles.get_sorted_organization_roles`. For consistency (with `get_default`, `get_top_dog`, etc.), we could keep a `RoleManager` method that delegates to `self.organization_roles`, though I'd rather not even do that.